java/tools/src/java/org/apache/orc/tools/FileDump.java

```diff
@@ -119,7 +119,7 @@ public static void main(Configuration conf, String[] args) throws Exception {
     }
 
     // if the specified path is directory, iterate through all files and print the file dump
-    List<String> filesInPath = new ArrayList<>();
```

Comment:
The same mistake like https://github.com/apache/orc/pull/1181#discussion_r924633593